### ISSUE TITLE
Increase EMA-parameter during training

### ISSUE THREAD

**Issue by Benjamin-Hansson** [lucidrains/byol-pytorch#82]
Hi, I noticed that the EMA-parameter (called beta in the code, tau in the paper) is not updated during training. In the paper they describe that they increase tau from the start value to 1 during training. This makes a huge difference to the validation loss at the end of the training. 

![without_tau_update](https://user-images.githubusercontent.com/61389429/168551312-d067e847-b653-419e-a2c1-515b76e264a8.png)
![with_tau_update](https://user-images.githubusercontent.com/61389429/168551316-2bfac0fe-fdca-43d2-b9a7-1009d16686ee.png)

**Comment by michal-choinski** [comment]
@Benjamin-Hansson just to clarify - when did you observe this spiking loss? Was it when the EMA parameter was not updated, or the other way around?
